src/io/flutter/sdk/FlutterSdk.java

```diff
@@ -244,7 +244,7 @@ public FlutterCommand flutterRun(@NotNull PubRoot root,
       args.add("--device-id=" + device.deviceId());
     }
 
-    if (mode == RunMode.DEBUG) {
+    if (mode == RunMode.DEBUG || mode == RunMode.RUN) {
```

Comment:
is there any other mode? also make sure we are not breaking release/profile mode.

Reply:
There is also `RunMode.PROFILE` - I guess it would make sense to enable --start-paused for all of the IntelliJ modes (so in that case just remove this conditional)?

I'm having trouble testing flutter profile mode and release mode. Do I need to go through some additional setup steps for this? I se this doc for iOS at least https://flutter.dev/docs/deployment/ios